config/_default/menus/main.en.yaml

```diff
@@ -5125,7 +5125,7 @@ menu:
       parent: appsec_enabling_single_step
       identifier: appsec_enabling_single_step_code
       weight: 2102
-    - name: Using Datadog Tracing libraries
+    - name: Using Datadog Tracing Libraries
```

Comment:
We don't want to change this one?
```suggestion
    - name: Using Datadog APM SDKs
```

Reply:
Thanks for catching. This should be fixed now.